crm_lead_group_permits/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+#
+#    SDi Soluciones Informáticas
+#    Copyright (C) 2020-Today SDi Soluciones Digitales <www.sdi.es>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'CRM_LEAD_GROUP_PERMITS',
+    'summary': 'Creation of a group that gives permits to create an opportunity',
```

Comment:
Añade en el summario: only these group can do it, anybody else don't it.

Para remarcar que estamos quitando permisos al grupo de ventas "documentos propios".